src/main/webapp/app/shared/sidebar/sidebar.component.ts

```diff
@@ -33,6 +36,7 @@ export class SidebarComponent implements OnDestroy, OnChanges, OnInit {
     @Input() channelTypeIcon?: ChannelTypeIcons;
     @Input() collapseState: CollapseState;
     @Input() showFilter: boolean = false;
+    @Input() inCommunication: boolean = false;
 
```

Comment:
here same for the input

```suggestion
    inCommunication = input<boolean>(false);
```